Description: Disable vusage.conf since we don't install vusaged at all.
 Tracked down by: Renato Botelho <garga@FreeBSD.org>
Forwarded: not-needed
Author: Peter Pentchev <roam@FreeBSD.org>
Last-Update: 2010-02-09

--- a/vusagec.conf
+++ b/vusagec.conf
@@ -23,7 +23,7 @@
 	  True or False
    */
 
-   Disable = False;
+   Disable = True;
 
    /*
 	  Daemon is listening on local socket
